src/main/java/org/rassee/omniture/pig/AbstractSchemaGenerator.java

```diff
@@ -39,8 +39,8 @@
 public abstract class AbstractSchemaGenerator implements SchemaGenerator {
     //A simple Set based lookup that simply defines columns for each data types
     //It is important to use right data type if we know for sure. This offers efficient storage of output in parquet or orc format
-    private final static Set<String> timestampColumns = Sets.newHashSet("date_time");
-    private final static Set<String> longColumns = Sets.newHashSet("browser_height", "browser_width", "search_engine",
+    private static final Set<String> timestampColumns = Sets.newHashSet("date_time");
+    private static final Set<String> longColumns = Sets.newHashSet("browser_height", "browser_width", "search_engine",
```

Comment:
@msukmanowsky It is just java convention to use modifiers in this order. It does not effect any logic.